src/main/topLevelPages/myDashboard/newVirtualContributorWizard/useNewVirtualContributorWizard.tsx

```
@@ -405,23 +425,56 @@ const useNewVirtualContributorWizard = (): useNewVirtualContributorWizardProvide
-      const callout = await handleCreateCallout(calloutDetails);
+    if (calloutData?.posts && calloutData?.posts.length > 0 && bokId) {
+      const postCallout = await handleCreateCallout(calloutDetails);
+      const postCalloutId = postCallout?.id;
+      calloutId.current = postCalloutId;
 
       // add posts to collection
-      if (callout?.id) {
-        for (const post of calloutPostData?.posts) {
-          await onCreatePost(post, callout.id);
+      if (postCalloutId) {
+        for (const post of calloutData?.posts) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)